VTUL/dlp-access
# Adds lang field to site table and uses that value to fetch language config files

**Adds lang field to site table and uses that value to fetch language config files.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2377) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Adds lang field to site table and uses that value to fetch language config files

# What's the changes? (:star:)
* Adds lang field to the site data model
* Uses the value in lang field to fetch language configuration files from s3 and then caches them in browser's session storage.

# How should this be tested?
* Visit `/search` page
* Check that items/collections can still be filtered using the language facet checkboxes in the left column

# Additional Notes:
* branch: `LIBTD-2377`
* Site table will need to be copied from `langconfig` amplify env after merge

# Interested parties
@yinlinchen 

(:star:) Required fields
